src/artemis/fast_grid_scan_plan.py

```diff
@@ -91,7 +98,9 @@ def get_plan(parameters: FullParameters):
     )
     zebra = Zebra(name="zebra", prefix=f"{parameters.beamline}-EA-ZEBRA-01:")
 
-    return run_gridscan(fast_grid_scan, zebra, eiger, parameters)
+    undulator = Undulator(name="undulator", prefix=f"{parameters.beamline}-MO-SERVC-01:")
```

Comment:
**must**: This isn't quite going to work. The beamline prefix is `BL03I` but the prefix for the undulator is `SR03I`. I think to solve this we need a new parameter called something like `insertion_prefix` that GDA will pass in

Reply:
`beamline` replaced with new parameter `insertion_prefix`